exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Transaction.kt

```diff
@@ -39,6 +39,9 @@ open class Transaction(private val transactionImpl: TransactionInterface) : User
     var duration: Long = 0
     var warnLongQueriesDuration: Long? = db.config.warnLongQueriesDuration
     var debug = false
+    var repetitionAttempts: Int = 0
+    var minRepetitionDelay: Long = 0
+    var maxRepetitionDelay: Long = 0
```

Comment:
These values are also now accessible within a regular `transaction`, even though they won't actually do anything:
```kt
transaction(repetitionAttempts = 0) {
    repetitionAttempts = 5  // this does nothing & block will not repeat
}
```
Given that this may be misleading, should this duplicate be addressed in this PR or in a new one for `transaction`?